modules/ob-installing-buildpacks-v3-strategy.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: PROCEDURE
+[id="installing-buildpacks-v3-strategy_{context}"]
+= Installing buildpacks v3 strategy
+
+You can install the `buildpacks v3` strategy at your namepsace or cluster level. By installing the `buildpacks v3` strategy at cluster level, you can share it across different namespaces within your cluster.
+
+.Prerequisites
+
+* You have installed the `oc` CLI.
+
+.Procedure
+
+* To install the `buildpacks v3` strategy at cluster level, run the following command:
++
+[source,terminal]
+----
+oc apply -f samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3-heroku_cr.yaml
```

Comment:
I think, the following would be easier, as it won't make cloning the repository mandatory in he local env for any end-user:
```suggestion
oc apply -f https://raw.githubusercontent.com/shipwright-io/build/main/samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3-heroku_cr.yaml
```

However, I haven't seen this pattern in our official docs yet. @shipsing let me know in case this does not abide by the documentation format.